replayer/zstdstream.h

```diff
@@ -21,6 +21,11 @@
 #define ZSTD_STATIC_LINKING_ONLY
 #include <zstd.h>
 
+#ifdef _MSC_VER
+#include <BaseTsd.h>
+typedef SSIZE_T ssize_t;
```

Comment:
Does this work because MSVC defines SSIZE_T? Briefly looked it up but didn't find anything.

Reply:
Yup, see https://msdn.microsoft.com/en-us/library/windows/desktop/aa383751(v=vs.85).aspx#SSIZE_T